client/src/controllers/DrilldownController.ts

```diff
@@ -1,10 +1,21 @@
 import { Controller } from '@hotwired/stimulus';
-import { ActionController } from './ActionController';
-import { DropdownController } from './DropdownController';
+import type { ActionController } from './ActionController';
+import type { DropdownController } from './DropdownController';
 
 /**
  * Drilldown menu interaction combined with URL-driven
  * state management for listing filters.
+ * 
+ * @example
+ * <section ><div data-controller="w-drilldown">
```

Comment:
```suggestion
 * ```html
 * <section ><div data-controller="w-drilldown">
```

Can you wrap this in three backticks, like markdown. 

We don't do this consistently yet but it's nice for new examples.